src/app/games/apagos/ApagosTutorial.ts

```diff
@@ -0,0 +1,47 @@
+import { ApagosMove } from 'src/app/games/apagos/ApagosMove';
+import { ApagosState } from 'src/app/games/apagos/ApagosState';
+import { TutorialStep } from 'src/app/components/wrapper-components/tutorial-game-wrapper/TutorialStep';
+import { ApagosCoord } from './ApagosCoord';
+import { Player } from 'src/app/jscaip/Player';
+import { MGPValidation } from 'src/app/utils/MGPValidation';
+
+export class ApagosTutorial {
+
+    public tutorial: TutorialStep[] = [
+        TutorialStep.informational(
+            $localize`Initial board and goal of the game`,
+            $localize`At Apagos, there is 4 squares, each of them have a fix number of holes to put pieces, each player start with 10 pieces. Dark pieces belong to first player, light pieces belong to the second one. The game ends when no player can do a move, the player owning the most pieces in the righter square wins!`,
+            ApagosState.getInitialState(),
+        ),
+        TutorialStep.anyMove(
+            $localize`Drop`,
+            $localize`One of the two kind of move is the drop. To do one, you must click on any visible arrow, being of your color or the opponent's. If the receiving square is one of the 3 lefter ones, it will 'climb' one step right, and the one on its right will slide one step below. You play Light.<br/><br/>Drop a piece on one of thoses three squares.`,
```

Comment:
"One possible kind of move is a drop" ça veut pas dire "un type de mouvement possible est un drop" c'qui serait dégueux ? En plus pourquoi ne pas préciser directement que y en a que deux ?

Reply:
Of any color, je dirais que non, déjà avec le tuto disant "la tienne ou celle de l'adversaire" ça t'as semblé perturbant, je sais pas si être moins expressément exhaustif aiderait, non ?
Targetted square du coup ?
"it will switch its coord with the square right to it" ?

Reply:
Alors: "There are two kind of moves. One is the drop." Phrases simples au max dans un tuto, une info à la fois :D 
alors je mettrais "on a visible arrow, being ..."
if the target square is = si la case cible est…
switch its coord: non on parle pas de coordonnées dans un tuto il me semble :D Sinon: it will exchange position with the square *on its right*

Reply:
ça fait bizarre de parler de case cible, case ciblée me semble plus normal, ça fait moins "on rajoute des termes techniques pour faire genre" non ?

Reply:
pourquoi pas simplement "la case choisie" (the chosen square) ?

Reply:
fait!